src/device/aarch64/asm.go

```diff
@@ -0,0 +1,42 @@
+// +build aarch64
```

Comment:
I'm thinking that this build tag `aarch64` should be renamed to `arm64` everywhere, to match what Go does (`GOARCH=arm64`). And maybe also the package name and path.